test/test.ref

```diff
@@ -0,0 +1,142 @@
+# 1 "test.cppo"
+(* comment *)
+
+# 4 "test.cppo"
+f(1)
+# 6 "test.cppo"
+ 2+ 3.14  
```

Comment:
Are the spaces around `3.14` expected?
Context:
https://github.com/ocaml-community/cppo/blob/08b6c59eb53b688d2b66a16c5fa70097918789c0/test/test.cppo#L3-L6

Reply:
Without looking at the source code, I would say that the reason to surround expanded arguments with spaces was to avoid "token squashing". The goal is preserve the original lexical units and avoid accidentally fusing them. For example, one could write:
```
#define plus +
x+plus
```
^ The above should expand to `x+ +`, which is equivalent to `x + +`, but not the same as `x++` or `x ++`. We want to avoid the latter, where `++` has become a single OCaml token by accident.